docs/tutorials/geomesa-quickstart-storm.rst

```diff
@@ -272,7 +272,7 @@ same location.
 
 .. note::
 
-    The heatmap style requires that ``geomesa-process-wps`` be installed in your
```

Comment:
i think this was actually right - geomesa-process will be installed as part of the regular gs-plugin, but you need to also install geomesa-process-wps